wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,109 @@
+# Installing and Updating drivers 
+In this guide, you will learn how to download or update drivers on your computer.
+
+::: warning
+It is highly recommended to avoid using third-party software to update or install your drivers.
+:::
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager. You can access the device manager by right-clicking the Windows start button or by pressing <kbd>Win</kbd> + <kbd>X</kbd> and selecting Device Manager. In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Using Windows Update
+You can usually find drivers for your devices in Windows Update, instead of having to manually download the file.
+ 1. Open Settings through the start menu or by pressing <kbd>Win</kbd> + <kbd>I</kbd>.
+ 2. Go to Updates and Security or Windows Updates.
+ 3. Click on “Check for Updates” or "Download & Install Updates".
+ 4. Windows Update might say that you are up-to-date, even if optional updates are available. You should check if there are additional optional updates by clicking on "View Additional Updates" on Windows 10, or "Advanced Options"> "Optional Updates" on Windows 11. Doing so will allow you to view available driver updates/software updates. Select all of them and click on “Download and install”.
+<img src="./wiki/img/Installing-drivers-and-fix-them/Optional updates/Windows 11
+/1.png" alt="4.1" height=350px>
+<img src="./wiki/img/Installing-drivers-and-fix-them/Optional updates/Windows 11
+/2.png" alt="4.2" height=350px>
+<img src="./wiki/img/Installing-drivers-and-fix-them/Optional updates/Windows 11
+/3.png" alt="4.3" height=350px>
```

Comment:
You cannot have newlines in the middle of strings—this is why the checks failed. The paths also do not need to start with `./wiki` because you are already in the `wiki` folder. They should start with `./img`.
```suggestion
<img src="./img/Installing-drivers-and-fix-them/Optional updates/Windows 11/1.png" alt="4.1" height=350px>
<img src="./img/Installing-drivers-and-fix-them/Optional updates/Windows 11/2.png" alt="4.2" height=350px>
<img src="./img/Installing-drivers-and-fix-them/Optional updates/Windows 11/3.png" alt="4.3" height=350px>
```